### ISSUE TITLE
BatchNormalization mode=2

### ISSUE THREAD

**Issue by divamgupta** [keras-team/keras#11755]
How to get BatchNormalization mode=2 which was in the previous versions of keras?
According to the documentation of keras==1.2.2 mode=2 does :  ' feature-wise normalization, like mode 0, but using per-batch statistics to normalize the data during both testing and training.'

Why was mode removed? and is there any way to get that in the latest version of keras.

**Comment by harshini-gadige** [comment]
Hi!
This issue isn't related to a bug/enhancement/feature request or other accepted types of issue.

Request you to post these type of questions in the following resources :

https://keras.io/
https://keras.io/#support
https://gitter.im/Keras-io/Lobby
Thanks!

If you think I made a mistake, please open another issue.
